src/custom/pages/Profile/LockedGnoVesting/hooks.ts

```diff
@@ -0,0 +1,129 @@
+import { useCallback, useEffect, useMemo, useState } from 'react'
+import { ContractTransaction } from '@ethersproject/contracts'
+import { CurrencyAmount, Token } from '@uniswap/sdk-core'
+import { useActiveWeb3React } from 'hooks/web3'
+import MERKLE_DROP_ABI from 'abis/MerkleDrop.json'
+import TOKEN_DISTRO_ABI from 'abis/TokenDistro.json'
+import { MerkleDrop, TokenDistro } from 'abis/types'
+import { useSingleCallResult } from 'lib/hooks/multicall'
+import { useTransactionAdder } from 'state/enhancedTransactions/hooks'
+import { useContract } from 'hooks/useContract'
+import { COW as COW_TOKENS } from 'constants/tokens'
+import { SupportedChainId } from 'constants/chains'
+import { OperationType } from 'components/TransactionConfirmationModal'
+import { fetchClaim } from './claimData'
+
+// We just generally use the mainnet version. We don't read from the contract anyways so the address doesn't matter
+const COW = COW_TOKENS[SupportedChainId.MAINNET]
```

Comment:
Are you sure this won't be a problem?
The chainId is available where this is used so I see no reason to not use the proper one per network and avoid unforeseen consequences

Reply:
We only use it to format currency amounts, since the number of decimals is generally the same this is safe. (It's just easier like this as we don't need to make it reactive to network changes.)